src/main/scala/cromwell/Main.scala

```diff
@@ -48,15 +50,24 @@ object Main extends App {
 
   def run(args: Array[String]): Unit = {
     if (args.length != 2) usageAndExit()
-    val runner = new SingleWorkflowRunner
-    val outputs = runner.run(new File(args(0)), new File(args(1)))
-    outputs match {
-      case Success(o) =>
-        import cromwell.binding.values.WdlValueJsonFormatter._
-        println("Workflow Completed.  Outputs are:")
-        println(o.toJson.prettyPrint)
-        runner.workflowManagerActor ! WorkflowManagerActor.Shutdown
-      case Failure(f) => println(f.printStackTrace())
+
+    try {
+      val wdl = FileUtil.slurp(Paths.get(args(0)))
+      val jsValue = FileUtil.slurp(Paths.get(args(1))).parseJson
+
+      val inputs: binding.WorkflowRawInputs = jsValue match {
+        case JsObject(rawInputs) => rawInputs
+        case _ => throw new RuntimeException("Expecting a JSON object")
+      }
+
+      val workflowManagerSystem = new WorkflowManagerSystem {}
+      val singleWorkflowRunner = SingleWorkflowRunnerActor.props(wdl, inputs, workflowManagerSystem.workflowManagerActor)
+      val actor = workflowManagerSystem.actorSystem.actorOf(singleWorkflowRunner)
+      // And now we just wait for the magic to happen
```

Comment:
Then you're right back where you started w/ not knowing when something is done. It's not insurmountable but that was already viewed as an issue.

I went back and forth on this myself and convinced myself that there's no benefit to having the actor communicate back to main. That said, there might be benefit in not having the actor automatically print its outputs and exiting. Perhaps the actor could be told "when you're done, go ahead and print to console and exit".

But if it's *not* exiting once complete, what does it do? Just hang out? Hang out until *when* exactly? Perhaps just until outputs are retrieved or it prints them out?